translations_auto_pr.js

```diff
@@ -130,10 +130,7 @@ console.log(`pushing to remove branch (git push -f origin ${branchName})`);
 cp.execSync(`git push -f origin ${branchName}`);
 
 console.log("Checking if there is already a pull request...");
-const octokit = new Octokit({auth: {
-    username: authUser,
-    password: authToken}
-});
+const octokit = new Octokit.Octokit({auth: authToken});
```

Comment:
This is the fix I made...looks like that change got bundled into this one when I ran the build to test the change.

Reply:
Fixes a regression from updating the octokit version in https://github.com/microsoft/vscode-cmake-tools/pull/1475